pytac/element.py

```diff
@@ -74,6 +76,11 @@ def cell(self):
         else:
             return int(self.s / self._lattice.cell_length) + 1
 
+    @property
+    def families(self):
```

Comment:
Do we want to start adding type hints for changed lines?

Reply:
Let's explicitly drop Python 2 first, I will do that today.